backend/de.metas.adempiere.adempiere/base/src/main/java-gen/org/compiere/model/I_C_Order.java

```diff
@@ -8,6 +8,10 @@
 public interface I_C_Order 
 {
 
+	String COLUMNNAME_AD_InputDataSource_ID = "AD_InputDataSource_ID";
```

Comment:
that's OK, but then please remove this stuff from `de.metas.order.model.I_C_Order`